CONTRIBUTING-PYTHON.md

```diff
@@ -23,8 +24,9 @@ poetry --version
 ```
 
 If the versions are not correct or you don't have Python or Poetry installed, download and follow their setup instructions:
-* Python: install with [pyenv](https://github.com/pyenv/pyenv)
-* Poetry: follow the official [Poetry installation instructions](https://python-poetry.org/docs/#installation)
+
```

Comment:
These changes, through to the addition of the new Changelog section at the bottom, are all formatting fixes